apps/smbmount.sh

```diff
@@ -35,7 +36,7 @@ fi
 
 # Inform the user
 msg_box "This script automates mounting SMB-shares locally in your system and adds them automatically as external storage to your Nextcloud."
-if [[ "no" == $(ask_yes_or_no "Do you want to continue?") ]]
+if ! yesno_box "Do you want to proceed with this script?"
```

Comment:
This is also something that we would change. The new implementation is better imo